gitolith-cli/Cargo.toml

```diff
@@ -14,3 +14,8 @@ edition = "2018"
 
 [dependencies]
 gitolith-core = { path = "../gitolith-core" }
+
+[dependencies.structopt]
+version = "0.3.21"
```

Comment:
Do we need the patch?

`version = "0.3"`

is enough imo.

Reply:
Fair enough.